src/main/java/frc/robot/constants/Ports.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.constants;
+
+public class Ports {
+
+    public enum JoystickPorts {
+        MAIN(0),
+        SECOND(1),
+        THIRD(2),
+        FOURTH(3),
+        FIFTH(4),
+        SIXTH(5);
+
+        private final int port;
+
+        JoystickPorts(int port) {
+            this.port = port;
+        }
+
+        public int getPort() {
+            return port;
+        }
+    }
```

Comment:
maybe move to joysticks package, this class was meant to help knowing the ids of motors and sensors between subsystems to not use same id and stuff like this and because joystickport is by itself maybe can move?